### PR TITLE
Update Description for isolatedModules flag

### PR DESCRIPTION
**Pull request by yuit**

### CODE REVIEW

File: pages/Compiler Options.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Option                                         | Type      | Default
 `--inlineSourceMap`                            | `boolean` | `false`                        | Emit a single file with source maps instead of having a separate file.
 `--inlineSources`                              | `boolean` | `false`                        | Emit the source alongside the sourcemaps within a single file; requires `--inlineSourceMap` or `--sourceMap` to be set.
 `--init`                                       |           |                                | Initializes a TypeScript project and creates a `tsconfig.json` file.
-`--isolatedModules`                            | `boolean` | `false`                        | Unconditionally emit imports for unresolved files.
+`--isolatedModules`                            | `boolean` | `false`                        | Transpile each file separately (similar to ts.transpile).
 `--jsx`                                        | `string`  | `"Preserve"`                   | Support JSX in `.tsx` files: `"React"` or `"Preserve"`. See [JSX](./JSX.md).
```
Comment (mhegazy) [inline pages/Compiler Options.md]: As a separate module

Comment (mhegazy) [inline pages/Compiler Options.md]: Also use 'transpileModule' instead of transpile

Comment (yuit) [inline pages/Compiler Options.md]: Should it be word like "Treat each file as a separate module and perform 'transpileModule' on each one (similar to ts.transpile)."

Comment (mhegazy) [inline pages/Compiler Options.md]: `Transpile each file as a separate module (similar to "ts.transpileModule")`

**Pull request merged**